src/main/kotlin/net/maizegenetics/phgv2/pathing/AlignmentUtils.kt

```diff
@@ -454,6 +460,29 @@ class AlignmentUtils {
             }
         }
 
+        /**
+         * Function that builds a set of minHashes from a sequence.
+         * This will work by doing a single pass over the read sequence and building the kmer hash set.
+         */
+        fun extractMinHash(sequence: String) : Set<Long> {
```

Comment:
Is this being called somewhere?

Reply:
Not yet.  I am using it for generating my training dataset for my Kmer Model but it was sitting in the same file as the other change so it was easier to just commit.  If it does not pan out I will remove it later.